src/test/ui/reject-specialized-drops-8142.rs

```diff
@@ -1,70 +1,152 @@
 // Issue 8142: Test that Drop impls cannot be specialized beyond the
 // predicates attached to the type definition itself.
 
-trait Bound { fn foo(&self) { } }
```

Comment:
@Mark-Simulacrum I am a bit confused, are tests not required to be rustfmt'd?
Could @oli-obk avoid reformatting this file and just tweak it slightly?

Reply:
Tests are currently not required to be rust formatted (we didn't want to deal with pretty-printing tests, primarily).

Reply:
In that case, I would prefer if this test remained mostly unchanged, as it would significantly reduce the diff.

Reply:
removed the rustfmt noise